### PR TITLE
Pinned components-contrib RC + Updated some deps

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
Updated some dependencies, also to make dependabot happy

Two dependencies that were updated required some small code changes

### CODE REVIEW

**Comment by berndverst** [comment]
Why not wait until new contrib is merged and then pin the new contrib?

**Comment by ItalyPaleAle** [comment]
> Why not wait until new contrib is merged and then pin the new contrib?

Done

Some deps here are unrelated from components-contrib

File: pkg/placement/raft/logger.go
Comment (mukundansundar) [inline pkg/placement/raft/logger.go:90]: why is this always returning info?

File: pkg/placement/raft/logger.go
Comment (mukundansundar) [inline pkg/placement/raft/logger.go:87]: should dapr debug level log also affect this?

File: pkg/placement/raft/logger.go
Comment (ItalyPaleAle) [reply pkg/placement/raft/logger.go:90]: This logger is kind of a nop and info is the default level

File: pkg/placement/raft/logger.go
Comment (ItalyPaleAle) [reply pkg/placement/raft/logger.go:87]: It's a nop

**Review by berndverst — APPROVED** [review]

**Pull request merged**
